src/views/mfa-verify/SecurityQuestionForm.js

```diff
@@ -10,7 +10,7 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-define(['okta', './inputs/ShowPasswordBox'], function (Okta, PasswordBox) {
+define(['okta'], function (Okta) {
```

Comment:
PasswordBox  is a courage component now https://github.com/okta/courage/blob/master/src/views/forms/inputs/PasswordBox.js
and we mapped all password type inputs to PasswordBox component
https://github.com/nikhilvenkatraman-okta/okta-signin-widget/blob/923487f3f10f49e354820a87f5c734bbdaac75f4/src/util/RegisterInputs.js#L12